diff --git a/include/js/dtlviewajax.js b/include/js/dtlviewajax.js
index 6afc799..d1281ec 100644
--- a/include/js/dtlviewajax.js
+++ b/include/js/dtlviewajax.js
@@ -262,6 +262,9 @@ function dtlViewAjaxSave(fieldLabel,module,uitype,tableName,fieldName,crmId)
 
 	var data = "file=DetailViewAjax&module=" + module + "&action=" + module + "Ajax&record=" + crmId+"&recordid=" + crmId ;
 	data = data + "&fldName=" + fieldName + "&fieldValue=" + escapeAll(tagValue) + "&ajxaction=DETAILVIEW"+groupurl;
+	if(module == 'ConfidentialInfo') {
+		data += "&cidwspinfo=" + encodeURIComponent(document.getElementById('cidwspinfo').value);
+	}
 	if(module == 'Users') {
 		data += "&form_token=" + (document.getElementsByName('form_token')[0].value);
 	}
